tools/lib/provision.py

```diff
@@ -37,7 +37,7 @@
 
 VAR_DIR_PATH = os.path.join(ZULIP_PATH, 'var')
 
-is_circleci = 'CIRCLECI' in os.environ
+is_ci = 'CI' in os.environ
```

Comment:
Let's call this `CONTINUOUS_INTEGRATION`; `CI` isn't unique enough of a name.

Reply:
(On the LHS; RHS presumably needs a more real check)

Reply:
We don’t really get a choice:

https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables

We could test for `os.environ.get("CI") == "true"` to be slightly more specific.

Reply:
Well, we could do an `||` of the values for two options.  But I suppose that check is probably OK.

Reply:
I guess being specific never hurts so I went with `'CIRCLECI' in os.environ or 'GITHUB_ACTIONS' in os.environ`.